content/projects.md

```diff
@@ -0,0 +1,21 @@
+# Projects
+
+## Organization
+
+TODO: Explain what is the minimal documentation required for any Optakt project.
+TODO: Explain what how we handle CI and releases with GitHub Actions.
+
+## Git Workflow
+
+TODO: Explain how we work with branches and why. Mention feature branches for big features.
+TODO: Explain that each PR has to be related to one issue.
+TODO: Explain PR requirements (labels, squash merges only, CI should pass, one approval at least, etc.)
```

Comment:
We also had the `could`, `should`, `must` classification of issues, perhaps it warrants a mention here. It was mostly during backlog refinement so not all issues necessarily had it